resources/inputs/register-form.tsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
```

Comment:
All this page is completely dulicated.

Is almost similar than the login component.

We can extract several partials to reduce the logic.